src/http/gateway/routes/index.js

```diff
@@ -1,3 +1,3 @@
 'use strict'
 
-module.exports = [require('./gateway')]
+module.exports = [...require('./gateway')]
```

Comment:
you dont need to spread here the default exports is alread an array
```suggestion
module.exports = require('./gateway')
```